src/utilities.h

```diff
@@ -870,4 +870,26 @@ size_t mathmlChildCount(const XmlNodePtr &node);
  */
 XmlNodePtr mathmlChildNode(const XmlNodePtr &node, size_t index);
 
+/**
+ * @brief Return the variables in the given model.
+ *
+ * Return the variables in the given model.
+ *
+ * @param model The model for which we want the variables.
+ *
+ * @return The variables in the given model.
+ */
+std::vector<AnalyserVariablePtr> variables(const AnalyserModelPtr &model);
```

Comment:
I really think this function (and the next) should be called `analyserVariables`. This function retuns a vector of analyser variables which in libCellML are a different concept from variables.

Reply:
Agreed, fixed in bbc20a9d8.